play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

```diff
@@ -59,16 +59,17 @@ Duration: 	2 Actions
 Range:		30m
 Effect:		You launch off three harmasist spikes in quick succession using both
 hands. Each has an 18 % chance to miss. You may pick up to three targets to 
-launch at. Each spike that hits deals 52 piercing damage. If target has no 
-shield left and is still alive, apply a poison payload of your choice.
+launch at. Each spike that hits deals 52 piercing damage and has an armor piercing 
+level of 3. If the needles defeat the armror of the target and the target is still alive, 
+apply a poison payload of your choice.
 
 /////	Peircing Spike
 Adrenaline Cost:35
 Needle Cost: 1
 Duration: 	2 Actions
 Range:		45m
-Effect:		10 % chance to miss. Deal 42 piercing damage.
-ignores sheilds and balistic armor. Apply a poison payload of your choice.
+Effect:		10 % chance to miss. Deal 42 piercing damage. AP level 4.
+if the  needle defeats the target's armor, apply a poison payload of your choice.
```

Comment:
Now that I understand the purchase hierarchy for medic abilities (thanks for updating your image :) , is it your intention that Harmasist, Fast, and Piercing Spike should be roughly equal in power? Or should the abilities be more powerful higher up the tree they go?
Right now, it looks like they are more-or-less equal.

Reply:
wilco! :)